docs/publishing-your-site.md

```diff
@@ -142,6 +142,14 @@ the following command from the directory containing the `mkdocs.yml` file:
 mkdocs gh-deploy --force
 ```
 
+This will build your documentation and deploy it to a branch
```

Comment:
Hmm, why are there remnants here? Did you merge the latest changes in mater into your fork?